java/client/src/main/java/glide/api/models/commands/SortStandaloneOptions.java

```diff
@@ -0,0 +1,30 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+@Builder
+public class SortStandaloneOptions {
```

Comment:
should this instead be a child class to sortoptions that has everything sort has while adding the standalone only ones? feels like that would save a few extra signatures

Reply:
That was my initial approach and unfortunately it didn't work because it is for standalone clients and it needs it's own parameter. And if it's a child class, it could still be used by cluster `sort(SortOptions sortOptions)` requests which is not supported by redis currently.

Reply:
right, having them not in an inheritence rules out the possibility of a user passing the child class object around by mistake, i guess that is indeed one of the factors for our commands